packages/story-editor/src/components/library/panes/media/common/stories/mediaElement.js

```diff
@@ -176,28 +155,7 @@ _Video.args = {
   },
 };
 
-export const _Video_With_Attribution = (args) => {
-  const resource = args.resource;
-
-  return (
-    <SnackbarContext.Provider value={snackbarValue}>
-      <MediaContext.Provider value={mediaValue}>
-        <ApiContext.Provider value={apiValue}>
-          <CanvasProvider>
-            <Column>
-              <MediaElement
-                index={0}
-                resource={resource}
-                width={150}
-                onInsert={action('insert into canvas')}
-              />
-            </Column>
-          </CanvasProvider>
-        </ApiContext.Provider>
-      </MediaContext.Provider>
-    </SnackbarContext.Provider>
-  );
-};
+export const _Video_With_Attribution = _Image.bind({});
```

Comment:
🤔 what's this for? Is it needed so `args` can be passed in by storybook?

Reply:
yep. It  basically binds (copies) `_Image` so that I don't have to write the entire component again. Then we get a second story that is a copy of `Image`, but named `Video With Attribution` The nice part is we can redefine the values for the args below. So while the stories `Image` and `Video With Attribution` are the same component, they are using different values for the props and thus we get different views of the same component, without having to copy / paste just to change a prop.